dotnet/xml/ns-Microsoft.ML.ImageAnalytics.xml

```diff
@@ -1,6 +1,8 @@
 <Namespace Name="Microsoft.ML.ImageAnalytics">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Namespace containing image loading and <a href="https://docs.microsoft.com/en-us/dotnet/machine-learning/resources/glossary#transform">transformation</a> capabilites. 
+    It is contained in the <a href="https://www.nuget.org/packages?q=Microsoft.ML.ImageAnalytics">Microsoft.ML.ImageAnalytics nuget package.</a>
+  </summary>
```

Comment:
>   [](start = 0, length = 2)

please align the indentation

Reply:
Consider linking to nuget package directly instead of search query results

https://www.nuget.org/packages/Microsoft.ML.ImageAnalytics/